### PR TITLE
[No QA] Emphasize commit signing in contributing guide

### PR DESCRIPTION
**Pull request by roryabraham**
### Details
Emphasize that commits must be signed in contributing guide, and provide better instructions for how to do so.

### Fixed Issues
n/a

### Tests
n/a, markdown changes only.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,7 +25,18 @@ We are currently managing payment via Upwork. If you'd like to be paid for your
...
-1. [Open a PR](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Be sure to fill in all the required information on the PR template, and be sure all of your [commits are signed](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits).
+1. **Before you start writing any code**, please be aware that we require all commits to be [signed](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits). The easiest way to do that is to [generate a new GPG key](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-gpg-key) and [add it to your Github account](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-gpg-key-to-your-github-account). Once you've done that, you can automatically sign all your commits by adding the following to your `.gitconfig`:
+    ```
+    [commit]
+        gpgsign = true
+    [user]
+        email = <Your GH account email>
+        name = <Your Name>
+        signingke = <your_signing_key>
```
Comment (rafecolton) [inline CONTRIBUTING.md:35]: Should be `signingkey` (missing the `y`)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -25,7 +25,18 @@ We are currently managing payment via Upwork. If you'd like to be paid for your
 ## Submitting a Pull Request
 #### Proposing a Change
 1. Fork this repository and create a new branch
-1. [Open a PR](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Be sure to fill in all the required information on the PR template, and be sure all of your [commits are signed](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits).
+1. **Before you start writing any code**, please be aware that we require all commits to be [signed](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits). The easiest way to do that is to [generate a new GPG key](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-gpg-key) and [add it to your Github account](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-gpg-key-to-your-github-account). Once you've done that, you can automatically sign all your commits by adding the following to your `.gitconfig`:
```
Comment (rafecolton) [inline CONTRIBUTING.md:28]: NAB I would consider adding some emojis (e.g. 💥 to either side of this text) to really make it stand out.

**Review by rafecolton — CHANGES_REQUESTED** [review]
This needs to be added to our other OSS repos, yeah?

File: CONTRIBUTING.md (hunk 2 above)
Comment (roryabraham) [reply CONTRIBUTING.md]: Went with ‼️

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -25,7 +25,18 @@ We are currently managing payment via Upwork. If you'd like to be paid for your
 ## Submitting a Pull Request
 #### Proposing a Change
 1. Fork this repository and create a new branch
-1. [Open a PR](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Be sure to fill in all the required information on the PR template, and be sure all of your [commits are signed](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits).
+1. ‼️ **Before you start writing any code** ️‼️, please be aware that we require all commits to be [signed](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits). The easiest way to do that is to [generate a new GPG key](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-gpg-key) and [add it to your Github account](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-gpg-key-to-your-github-account). Once you've done that, you can automatically sign all your commits by adding the following to your `.gitconfig`:
```
Comment (rafecolton) [inline CONTRIBUTING.md:28]: What kind of magic is this that these automatically become emojis?

**Review by rafecolton — APPROVED** [review]
:shipit:

File: CONTRIBUTING.md (hunk 3 above)
Comment (roryabraham) [reply CONTRIBUTING.md:28]: !! !== ‼️

**Review by Jag96 — APPROVED** [review]

**Pull request merged**